cime_config/cesm/machines/Makefile

```diff
@@ -291,14 +288,7 @@ endif
 #===============================================================================
 # Set include paths (needed after override for any model specific builds below)
 #===============================================================================
-INCLDIR += -I$(SHAREDPATH)/include -I$(SHAREDPATH)/$(COMP_INTERFACE)/$(ESMFDIR)/$(NINST_VALUE)/csm_share
-
-#
-#  Tell gmake to look in the csm_share directory for shr_*.mod files
-#
-
-vpath  shr%.mod $(SHAREDPATH)/$(COMP_INTERFACE)/$(ESMFDIR)/$(NINST_VALUE)/csm_share
-
+INCLDIR += -I$(INSTALL_SHAREDPATH)/include -I$(INSTALL_SHAREDPATH)/$(COMP_INTERFACE)/$(ESMFDIR)/$(NINST_VALUE)/include
```

Comment:
In talking with Jim E, it sounds like things in the sharedlib area do get built against things in the case-specific INSTALL_SHAREDPATH, but this shouldn't cause a problem. (In particular, I wondered if the include path has any effect at runtime, and it sounds like the answer is No.) So I guess I'm okay with this.